internal/pg/doc.go

```diff
@@ -0,0 +1,20 @@
+// package pg is a collection of resources that interact with the various
+// pieces of software in the PostgreSQL ecosystem (PostgreSQL itself, pgBouncer,
+// pgBackRest etc.)
+
+package pg
```

Comment:
I don't like this package. `internal/postgres` seems fine to me.

Reply:
@cbandy Hm. Per the discussion we had, this is going to be used to group the things that require interacting amongst PG-related services, e.g.:

- postgres
- pgadmin
- pgbouncer
- pgbackrest

et al. Specifically, functionality that requires a direct interaction.

The issue at hand is that there are some things that may be mixed, e.g. pgBouncer leverages some of the pg/password stuff.

I think the grouping shows that there is a potential coupling between them, vs. having it at a higher level.

Reply:
Conceded this and moved the directory up a level.